kitsune/messages/jinja2/messages/inbox.html

```diff
@@ -1,53 +1,61 @@
 {% extends "messages/base.html" %}
 {% from "messages/includes/macros.html" import avatar_link, name_link %}
 {% set title = _('Inbox') %}
-{% set crumbs = [(None, title)] %}
+{% set crumbs = [(url('messages.inbox'), _('Messages')),
+                 (None, title)] %}
 {% set active_tab = 'inbox' %}
 
 {% block content %}
-  <div class="sumo-page-section">
+<div class="sumo-page-section">
     <article id="inbox" class="main">
       <h1 class="sumo-page-heading">{{ title }}</h1>
       <div class="actions sumo-button-wrap">
         <a class="sumo-button primary-button" href="{{ url('messages.new') }}">{{ _('New Message') }}</a>
       </div>
-
-      {% if msgs %}
-        <form action="{{ url('messages.bulk_action') }}" method="POST">
-          <div class="sumo-button-wrap">
-            <input class="sumo-button" type="submit" name="mark_read" value="{{ _('Mark Selected as Read') }}">
-            <input class="sumo-button" type="submit" name="delete" value="{{ _('Delete Selected') }}">
-          </div>
-
-          <ol class="message-list">
-            {% for message in msgs.object_list %}
-              <li
-                class="message-list--item
-                  {% if message.unread %}unread{% endif %}
-                  {% if message.replied %}replied{% endif %}"
-              >
-                <div class="field checkbox no-label">
-                  <input type="checkbox" name="id" value="{{ message.id }}" id="id_checkbox_{{ message.id }}">
-                  <label for="id_checkbox_{{ message.id }}"></label>
+    {% if msgs %}
+    <form action="{{ url('messages.bulk_action') }}" method="POST">
+      <div class="sumo-button-wrap">
+        <input class="sumo-button" type="submit" name="mark_read" value="{{ _('Mark Selected as Read') }}">
+        <input class="sumo-button" type="submit" name="delete" value="{{ _('Delete Selected') }}">
+      </div>
+      <div class="inbox-table">
+            <div class="email-row header">
+                <div class="email-cell checkbox">
+                    <input type="checkbox" aria-label="Select all">
+                </div>
+                <div class="email-cell avatar"></div>
+                <div class="email-cell from">{{ _('From User') }}</div>
```

Comment:
Since this will always be a user, I'm wondering if just `From` might be OK?

Reply:
I had some earlier feedback from @emilghittasv about consistency in the column headers, and I carried that over to the inbox from the outbox. Your point is good, though - I think I'll drop the "User" piece.